packages/cashier/src/Stores/__tests__/crypto-fiat-converter-store.spec.js

```diff
@@ -0,0 +1,323 @@
+        WS.send.mockResolvedValue({
+            exchange_rates: {
+                base_currency: 'USD',
+                date: 1650868989,
+                rates: {
+                    AED: 3.67,
+                },
+            },
+        });
+
+        const exchangeRate = await crypto_fiat_converter_store.getExchangeRate('USD', 'AED');
+
+        expect(exchangeRate).toEqual(3.67);
+    });
+
+    it('should call function validateWithdrawFromAmount when account transfer container and general store container are different', () => {
+        crypto_fiat_converter_store.validateFromAmount();
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.withdraw.validateWithdrawFromAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should call function validateTransferFromAmount when account transfer container and general store container are different', () => {
+        crypto_fiat_converter_store.root_store.modules.cashier.general_store.active_container = 'account_transfer';
+        crypto_fiat_converter_store.validateFromAmount();
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.account_transfer.validateTransferFromAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should call function validateWithdrawToAmount when account transfer container and general store container are different', () => {
+        crypto_fiat_converter_store.validateToAmount();
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.withdraw.validateWithdrawToAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should call function validateTransferToAmount when account transfer container and general store container are different', () => {
+        crypto_fiat_converter_store.root_store.modules.cashier.general_store.active_container = 'account_transfer';
+        crypto_fiat_converter_store.validateToAmount();
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.account_transfer.validateTransferToAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should reset converter details when onChangeConverterFromAmount is called without passing target value', async () => {
+        await crypto_fiat_converter_store.onChangeConverterFromAmount({ target: '' }, 'USD', 'AED');
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.percentageSelectorSelectionStatus
+        ).toBeCalledTimes(1);
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_to_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_from_error).toEqual('');
+        expect(crypto_fiat_converter_store.converter_to_error).toEqual('');
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeFalse();
+    });
+
+    it('should set converter_from_amount amount', async () => {
```

Comment:
@vinu-deriv also i believe that it will be better to change the description:
'should set converter_from_amount with an amount when onChangeConverterFromAmount method was called'